readme.md

```diff
@@ -1,149 +1,180 @@
-## GRAPE – Genomic relatedness detection pipeline
-### Description
+# GRAPE – Genomic relatedness detection pipeline
 
-The project intends to implement best-practices of estimation of recent shared ancestry in a production-ready way.
+## Description
 
-Support: https://t.me/joinchat/NsX48w4OzcpkNTRi
+GRAPE is an open-source end-to-end Genomic RelAtedness detection PipelinE.
+It requires a single multisamples VCF file as input and has a separate workflow for downloading and checking the integrity of reference datasets
+GRAPE incorporates comprehensive data preprocessing, quality control (QC), and several workflows for relatedness inference.
 
-Main features:
+<p align="center">
+    <img src="./scheme.png" alt="drawing" width="400"/>
+</p>
 
-1. It can handle input in hg37 and hg38.
-2. Implements phasing and imputation pipeline with GERMLINE + ERSA recent shared ancestry estimation
-3. It has a very fast alternative pipeline without phasing and imputation. It uses IBIS + ERSA.
-4. It has a special simulation workflow for the accuracy analysis.
-5. It is fully containerized in Docker.
-6. Fast pipeline workflow with `--flow ibis` option can process 2000 samples in a few minutes.
+## Installation
 
-### Stack
+```bash
+docker build -t genx_relatives:latest -f containers/snakemake/Dockerfile -m 8GB .
+```
 
-The pipeline is implemented with the Snakemake framework. All used components are wrapped in Singularity containers or isolated in a Conda environment.
-Pipeline can also be executed with dockstore (https://dockstore.org).
+The pipeline has three main steps:
+1. Downloading of the reference datasets.
+2. Quality control and data preprocessing.
+3. Application of the relationship inference workflow.
 
-The visualisation of execution graph: [svg](https://raw.githubusercontent.com/genxnetwork/grape/master/dag.svg).
+The pipeline is implemented with the Snakemake framework and can be accessible through the `snakemake` command.
+All the pipeline functionality is embedded into the GRAPE launcher: `launcher.py`, that invokes Snakemake under the hood.
+By default, all the commands just check the accessibility of the data and build the computational graph.
+To actually perform computation one should add `--real-run` flag to all the commands.
 
-Multi-core parallelization is highly utilized due to the ability to split input data by each sample/chromosome.
+## Downloading of the Reference Datasets
 
-Information about stages:
+This step only needs to be done once.
+To download reference dataset one should use `reference` command of the pipeline launcher.
+The command below downloads needed references to the directory specified by the flag `--ref-directory`.
+After that `--ref-directory` argument should now be used in all subsequent commands.
 
-**Germline workflow**
+```bash
+docker run --rm --privileged -it -v /media:/media -v /etc/localtime:/etc/localtime:ro \
```

Comment:
We can remove --priveleged flag. It was added for using singularity inside docker